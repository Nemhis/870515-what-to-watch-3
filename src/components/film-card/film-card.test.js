import React from 'react';
import renderer from 'react-test-renderer';

import FilmCard from './film-card';


const film = {
  name: `test name`,
  picture: `/picture/src.jpg`,
};

it(`Render correctly <FilmCard>`, () => {
  const tree = renderer
    .create(
        <FilmCard
          film={film}
          onMouseEnter={() => {}}
          onFilmClick={() => {}}
        />
    ).toJSON(`test`);

  expect(tree).toMatchSnapshot();
});
