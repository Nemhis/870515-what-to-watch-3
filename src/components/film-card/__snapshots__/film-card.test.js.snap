// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly <FilmCard> 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="test name"
      height="175"
      src="/picture/src.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="#"
      onClick={[Function]}
    >
      test name
    </a>
  </h3>
</article>
`;
